package.json

```diff
@@ -37,8 +56,37 @@
     "nuxt": "^2.14.12",
     "nuxtron": "^0.3.1",
     "sass-loader": "^10.1.1",
+    "vue-cli-plugin-electron-builder": "~2.0.0-rc.5",
+    "vue-jest": "^3.0.7",
     "vue-template-compiler": "^2.6.12",
-    "webpack": "^4.44.2"
+    "webpack": "^4.46.0"
+  },
+  "eslintConfig": {
```

Comment:
Doesn't this live in `.eslintrc.yml` now?

Reply:
This branch takes us back in time to a point where `.eslintrc.yml` doesn't exist.

Reply:
Yes, but that's why it needed to be rebased :)

Reply:
rebase missed that  😕